Kernel/MiniStdLib.cpp

```diff
@@ -10,31 +10,38 @@ extern "C" {
 
 void* memcpy(void* dest_ptr, const void* src_ptr, size_t n)
 {
+#if ARCH(I386) || ARCH(X86_64)
     size_t dest = (size_t)dest_ptr;
     size_t src = (size_t)src_ptr;
     // FIXME: Support starting at an unaligned address.
     if (!(dest & 0x3) && !(src & 0x3) && n >= 12) {
         size_t size_ts = n / sizeof(size_t);
-#if ARCH(I386)
+#    if ARCH(I386)
         asm volatile(
             "rep movsl\n"
             : "=S"(src), "=D"(dest)
             : "S"(src), "D"(dest), "c"(size_ts)
             : "memory");
-#else
+#    else
         asm volatile(
             "rep movsq\n"
             : "=S"(src), "=D"(dest)
             : "S"(src), "D"(dest), "c"(size_ts)
             : "memory");
-#endif
+#    endif
         n -= size_ts * sizeof(size_t);
         if (n == 0)
             return dest_ptr;
     }
     asm volatile(
         "rep movsb\n" ::"S"(src), "D"(dest), "c"(n)
         : "memory");
+#else
+    u8* pd = (u8*)dest_ptr;
+    const u8* ps = (const u8*)src_ptr;
```

Comment:
east const pls

```suggestion
    u8 const* ps = (u8 const*)src_ptr;
```

Reply:
The whole file uses west const. Is it really policy to make files self-inconsistent?

Reply:
yes, current policy is to make all new code use east const (and optionally update stuff around it, e.g. when changing a function signature), event if other things in that same file still use west const. One day the [clang-format folks will finish their discussion](https://reviews.llvm.org/D69764) and we can update all the remaining uses of west const in one go :)

Reply:
Done. But if we're going to reformat everything in one fell swoop eventually, is it really worth to:
- modify locally copied-code from west-const to east-const
- spend review time on this

?